docs/src/main/tut/index.md

```diff
@@ -0,0 +1,127 @@
+
+In order to achieve the microsite creation, it uses directly these plugins and libraries:
+
+* [tut-plugin](https://github.com/tpolecat/tut)
+* [sbt-site](https://github.com/sbt/sbt-site)
+* [sbt-ghpages](https://github.com/sbt/sbt-ghpages)
+* [scalatags](https://github.com/lihaoyi/scalatags)
+
+Additionally it's supported for other useful libraries and plugins like:
+
+* [sbt-native-packager](https://github.com/sbt/sbt-native-packager)
+* [sbt-scalafmt](https://github.com/olafurpg/scalafmt)
+* [sbt-pgp](https://github.com/sbt/sbt-pgp)
+* [sbt-header](https://github.com/sbt/sbt-header)
+
+## Prerequisites
+
+* [sbt](http://www.scala-sbt.org/) 0.13.8+
+* [jekyll](https://jekyllrb.com/)
+
+To satisfy the `jekyll` prerequisite, here a couple of hints for local and travis environments.
+
+### Local Environment
+
+Depending on your platform, you might do this with:
+
+```bash
+yum install jekyll
+
+apt-get install jekyll
+
+gem install jekyll
+```
+
+### Continuous Integration - Travis
+
+If you have enabled [Travis](https://travis-ci.org/) for your project, you might have to tweak some parts of your `.travis.yml` file:
+
+Potentially, your project is a scala project (`language: scala`), therefore you need to add the bundle gems vendro path in the `PATH` environment variable:
+
+```
+before_install:
+ - export PATH=${PATH}:./vendor/bundle
+```
+
+This is needed in order to install and be able to use `jekyll` gem from other parts of your project. We have to do it in the Travis `install` section:
+
+```
+install:
+ - ...
+ - gem install jekyll -v 2.5
+```
+
+## Setup
+
+First of all, add the following lines to the `project/plugins.sbt` file, within your project or sbt module where you want to use the `sbt-microsites` plugin. Depending on the version:
+
+Latest release:
+```
+addSbtPlugin("com.fortysevendeg"  % "sbt-microsites" % "0.1.0")
+```
+
+Latest snapshot built from the `master` branch code:
+
+```tut:evaluated
+println("""resolvers += Resolver.sonatypeRepo("snapshots")""")
+println(s"""addSbtPlugin("com.fortysevendeg"  % "sbt-microsites" % "${microsites.BuildInfo.version}"""")
+```
+
+Finally, to enable the plugin, add this to your `build.sbt` file:
+```
+enablePlugins(MicrositesPlugin)
+```
+
+## Build the microsite
+
+Once you have written down your documents you can build the microste running this sbt task:
+
+```
+sbt> makeMicrosite
+```
+
+Internally, sequentially it'll run other tasks, among theirs, [`tut`](https://github.com/tpolecat/tut) and `makeSite` ([sbt-site](https://github.com/sbt/sbt-site)) tasks.
+
+## View the microsite in your browser
+
+1. In a shell, navigate to the generated site directory in `target/site`
+
+2. Start jekyll with `jekyll serve`
+
+3. Navigate to http://localhost:4000/yourbase_url/ in your browser, where `yourbase_url` depends on your own preferences (see `micrositeBaseUrl` setting).
```

Comment:
Is it possible to have jekyll serve this at the root so people just go to localhost for a given site?